apps/prairielearn/src/sync/fromDisk/assessments.sql

```diff
@@ -23,6 +23,13 @@ FROM
   JSONB_ARRAY_ELEMENTS_TEXT($exam_uuids) AS exam_uuids;
 
 -- BLOCK get_imported_questions
+WITH
+  iqi AS (
+    SELECT
+      *
+    FROM
+      jsonb_to_recordset($imported_question_info::JSONB) AS (sharing_name text, qid text)
+  )
```

Comment:
I would be happy to see a more compact syntax for this if anyone knows a way, I tried doing it without the SELECT and I couldn't get it to work.

Reply:
https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/lib/manualGrading.sql#L350

Reply:
that's the syntax I was using before, but I couldn't get it working that way when I moved it in to the WITH clause. is it supposed to work that way in a with clause?

Reply:
Ah, I see. Yes, a with clause (CTE) must be a valid SQL command: https://www.postgresql.org/docs/current/queries-with.html